scripts/mkcloud

```diff
@@ -1404,7 +1416,7 @@ function sanity_checks()
 
 
 ## MAIN ##
-step_aliases="_new_admin _compute _upgrade _testupdate"
+step_aliases="blank_admin crowbar crowbar_admin updated_crowbar_admin updates nodes _upgrade _testupdate"
```

Comment:
Read 2 lines below this and you will see `$allcmds` already includes `$step_aliases` :-)